### ISSUE TITLE
make the api for scan and shell straight

### ISSUE THREAD

**Issue by IronMania** [mondoohq/client#181]
please see the fallowing commands

Scan a azure subscription
```
mondoo scan azure --subscription {subscriptionID}
```
Shell to azure subscription
```
mondoo shell -t az --option subscriptionID={subscriptionID}
```

please make the shell and scan commands work the same.
I Prefere the scan command, because it is more straight. I do not care if subscription ID is an option :-)

**Comment by atomic111** [comment]
@IronMania it is already planned

**Comment by atomic111** [comment]
@IronMania, we released mondoo version 6.15.0. The scan and shell commands work the same now.
